app/src/main/res/layout/layout_delete_confirmation.xml

```diff
@@ -0,0 +1,107 @@
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="@dimen/padding_30">
+
+    <TextView
+        android:id="@+id/sure_delete_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/no_dimen"
+        android:layout_marginTop="@dimen/no_dimen"
+        android:letterSpacing="0.1"
+        android:text="@string/sure_you_want_to_delete_this_target"
+        android:textAlignment="center"
+        android:textSize="@dimen/text_size_16"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageView
+        android:id="@+id/target_background_confirm_image_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_20"
+        android:src="@drawable/ic_oval_marker_bg"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/sure_delete_text_view" />
+
+    <ImageView
+        android:id="@+id/target_confirm_image_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintBottom_toBottomOf="@id/target_background_confirm_image_view"
+        app:layout_constraintEnd_toEndOf="@id/target_background_confirm_image_view"
+        app:layout_constraintStart_toStartOf="@id/target_background_confirm_image_view"
+        app:layout_constraintTop_toTopOf="@id/target_background_confirm_image_view"
+        app:srcCompat="@drawable/ic_chess" />
+
+    <TextView
+        android:id="@+id/target_title_confirm_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/no_dimen"
+        android:layout_marginTop="@dimen/margin_20"
+        android:layout_marginEnd="@dimen/no_dimen"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/target_background_confirm_image_view"
+        tools:text="Teo va a Camboya"/>
+
+    <TextView
+        android:id="@+id/warning_message_confirm_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/no_dimen"
+        android:layout_marginTop="@dimen/no_dimen"
+        android:layout_marginEnd="@dimen/no_dimen"
+        android:padding="@dimen/padding_30"
+        android:text="Remember that if you delete it, you won’t be able to chat with it’s matches anymore"
```

Comment:
Remember to add the texts to the strings file

Reply:
Thanks for your suggestion.

Reply:
Thanks for your suggestion